docs/page-models.md

```diff
@@ -5,4 +5,5 @@ We don't have a `BasePage` model that all page models inherit from - they inheri
 If creating new page models, make sure they also inherit from the following
 
 1. the `SocialFields` mixin, so that social image and text fields are added to the 'promote' tab.
-2. the `ColourThemeMixin`, so that a colour theme can be specified for the page.
+2. the `ColourThemeMixin`, so that a colour theme can be specified for the page. For more details on how the theme feature works, please see [this section](custom-features/theme.md) of the docs.
+3. the `ContactMixin`, so that a contact can be specified for the page's footer. For more details on how the contact feature works, please see [this section](custom-features/contact.md) of the docs.
```

Comment:
**Suggestion** (nitpick): Use meaningful link titles.

```suggestion
2. the `ColourThemeMixin`, so that a colour theme can be specified for the page. For more details on how the theme feature works, please see [Custom Features > Theme (docs)](custom-features/theme.md).
3. the `ContactMixin`, so that a contact can be specified for the page's footer. For more details on how the contact feature works, please see [Custom Features > Contact (docs)](custom-features/contact.md).
```

(Or wording that you like.)